openshift/s2i/bin/assemble

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+function is_django_installed() {
```

Comment:
Maybe switch this to checking for flask?

Reply:
I think you're misunderstanding what needs to be reviewed here. This is an upstream script.

We aren't forking the script to change everything about it.

We just want to patch the file to avoid installing known-bad versions of pip/pipenv.  Everything else about the script should follow the upstream version and avoid diverging as much as possible.

Reply:
Ah, with that in mind. I approve